comeet/AuthenticatorADALSettings.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  AuthenticatorADALSettings.swift
+//  comeet
+//
+//  Created by Kevin Burek on 4/22/17.
+//  Copyright © 2017 teamawesome. All rights reserved.
+//
+
+import Foundation
+
+class AuthenticatorADALSettings {
+    
+    internal struct Constants {
+        static let authority = "https://login.microsoftonline.com/common"
+        /// OAuth2 permission scopes:
+        /// Exchange Web Services permission:
+        ///
+        /// Outlook REST Api calendar read permission:
+        ///   https://msdn.microsoft.com/en-us/office/office365/api/calendar-rest-operations#GetEvent
+        static let resource = "https://outlook.office365.com https://outlook.office.com/calendars.read"
+        static let clientId = "a64d56ea-5675-4ccf-82e9-5757620e1d26"
+        static let redirectUri = URL(string: "http://localhost/comeet")
+        static let authCookies = ["MSISAuth", "MSISAuthenticated", "MSISLoopDetectionCookie"]
+    }
+    
+    static var authority = getAuthority()
+    
+    private class func getAuthority() -> String {
+        if let val = UserDefaults.standard.string(forKey: "oauth_authority") {
+            return val
+        }
+        return Constants.authority
+    }
+    
+    static var resource = getResource()
+    
+    private class func getResource() -> String {
+        if let val = UserDefaults.standard.string(forKey: "oauth_scope") {
+            return val
+        }
+        return Constants.resource
+    }
+    
+    static var clientId = getClientId()
+    
+    private class func getClientId() -> String {
+        if let val = UserDefaults.standard.string(forKey: "oauth_clientid") {
+            return val
+        }
+        return Constants.clientId
+    }
+    
+    static var redirectUri = getRedirectUri()
+    
+    private class func getRedirectUri() -> URL {
+        return Constants.redirectUri!
```

Comment:
We should try to avoid force unwrap. But this is tricky since URL (string:) returns an optional.